genshi/template/astutil.py

```diff
@@ -743,8 +743,16 @@ def _process_slice(node):
                 for dim in node.dims[1:]:
                     self._write(', ')
                     self.visit(dim)
+            elif isinstance(node, _ast.Call):
```

Comment:
I'm wondering if we can simply remove all of these cases and after `elif isinstance(node, _ast.Index):` write a catch all case of `else: self.visit(node)`.

Reply:
This does appear to be the case.  I'll do some more testing and update the PR if I don't identify any problems.